### DISCUSSION TITLE
Domain name for jqlang.github.io/jq/

### DISCUSSION THREAD

**Discussion by yochem** [stedolan/jq#3219] [Ideas]
Hi jq team, I was wondering if you'd considered buying a domain name to point to jqlang.github.io/jq/. I think this would look much better than the current one (and fixes the problem of jqlang.github.io not pointing to the actual website).

Here are some domain names of other programming languages:

Here's a list of homepages for several popular programming languages:

- https://www.python.org/
- https://www.javascript.com/
- https://www.rust-lang.org/
- https://go.dev/
- https://www.ruby-lang.org/
- https://www.php.net/
- https://www.swift.org/
- https://kotlinlang.org/
- https://www.typescriptlang.org/
- https://www.r-project.org/
- https://docs.microsoft.com/en-us/dotnet/csharp/
- https://www.scala-lang.org/
- https://julialang.org/
- https://www.haskell.org/
- https://www.erlang.org/
- https://clojure.org/
- https://www.lua.org/
- https://www.perl.org/
- https://www.gnu.org/software/bash/

Seems that '.org' is used most, and I also think it's a great fit for this project: https://jqlang.org. It's still available (and from what I saw, fairly cheap too at under $10/year).

Some advantages would be:
- Better SEO
- Professional look
- Don't have to use the `/jq/` path anymore
- Idk you can have @jqlang.org email addresses :)

**Comment by wader** [comment]
Seems like a good idea, if people agree i guess a maintainer should own/control it? i'm happy to take care of payment and manage it

**Reply by yochem** [reply]
Yeah me too, but it might be a good idea to not let this depend on a single person. For example, we could add the login for the domain provider to the organization's secrets (in a repo or with github secrets? Don't know what works best)

**Reply by itchyny** [reply]
I acquired the domain on Squarespace. If there is no objection from other maintainers, I will configure the custom domain of the Pages. I can invite @wader as a domain manager.

**Reply by wader** [reply]
@itchyny 👍 sounds good

**Reply by itchyny** [reply]
No objection was raised so far, but I'll wait for another week for feedback.

**Reply by owenthereal** [reply]
Can you please add @stedolan @nicowilliams and me to the domain as well?

**Reply by itchyny** [reply]
@owenthereal I invited you a few days ago but did you check the email box?

**Reply by owenthereal** [reply]
Thanks for sending the invite, @itchyny—I’m all set now!

I also started a discussion [here](https://github.com/jqlang/jq/discussions/3244) about moving jqplay to the jqlang org, following up on our Discord conversation.

**Comment by itchyny** [comment]
The website is now live at https://jqlang.org/! (#3243)
